resources/register-jaas-account.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+EMAIL=$1
+DOMAIN=$2
+
+if [ -z "${DOMAIN}" ] || [ -z "${EMAIL}" ]; then
+    echo "You need to provide email and domain as parameters."
+    exit 1
+fi
+
+JAAS_ENDPOINT="https://account-provisioning.cloudflare.jitsi.net/operations"
+CHALLENGE_FILE="/usr/share/jitsi-meet/jitsi-challenge.txt"
```

Comment:
I think the usual place to put this is in .well-known/

Reply:
This is where the service is looking for it?
@oanaianc @raluca8x8 ?

Reply:
.well-known is handled only for port 80 (https://github.com/jitsi/jitsi-meet/blob/779ecd6da640c9f2ff0393355f5cd97db290776b/doc/debian/jitsi-meet/jitsi-meet.example#L23), where the service requires the https to be in place ... but I guess with .txt it will work and under https

Reply:
But cannot change it till the service is updated.

Reply:
> This is where the service is looking for it? @oanaianc @raluca8x8 ?

yes, we check for the challenge in jitsi-chalenge.txt

Reply:
@saghul I got confirmation from @oanaianc that she will do the change to the service. So you can approve if you agree with the changes and I will merge it once I have the green light from @oanaianc .